translator-string/src/org/jetbrains/dukat/translatorString/StringTranslator.kt

```diff
@@ -341,16 +342,31 @@ private fun StatementModel.translate(): List<String> {
                 statements.flatMap { it.translate() }.map { FORMAT_TAB + it } +
                 listOf("}")
         is IfStatementModel -> {
-            val header = "if (${condition.translate()}) "
-            val mainBranch = thenStatement.translate()
-            val elseBranch = elseStatement?.translate() ?: listOf()
-            return listOf(header + mainBranch[0]) + mainBranch.drop(1) + elseBranch
+            val header = "if (${condition.translate()}) {"
+            val mainBranch = thenStatement.statements.flatMap { it.translate() }.map { FORMAT_TAB + it }
+            return when {
+                elseStatement == null -> listOf(header) + mainBranch + listOf("}")
+                elseStatement!!.statements.size == 1 && elseStatement!!.statements[0] is IfStatementModel -> {
```

Comment:
let's cast just once or at least reduce number of casts here
also, better to user firstOrNull